kytos/utils/napps.py

```diff
@@ -36,12 +36,24 @@ def __init__(self, controller=None):
         """
         self._controller = controller
         self._config = KytosConfig().config
-        self._installed = Path(self._config.get('napps', 'installed_path'))
-        self._enabled = Path(self._config.get('napps', 'enabled_path'))
+        self._kytos_api = self._config.get('kytos', 'api')
+        self._load_kytos_configuration()
 
         self.user = None
         self.napp = None
 
+    def _load_kytos_configuration(self):
+        """Request current configurations loaded by Kytos instance."""
+        uri = self._kytos_api + 'kytos/configuration/'
+        try:
+            options = json.loads(urllib.request.urlopen(uri).read())
+        except urllib.error.URLError:
+            print('Kytos is not running.')
+            sys.exit()
+
+        self._installed = Path(options.get('installed_napps'))
+        self._enabled = Path(options.get('napps'))
```

Comment:
Yes, it is. The controller.options have a field name 'napps' that store the Napps directory.